contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -832,3 +834,22 @@ drop_related_bbf_namespace_entries(int16 dbid)
 	table_endscan(scan);
 	table_close(namespace_rel, RowExclusiveLock);
 }
+
+const char *
+get_owner_of_db(const char *dbname)
+{
+	char				*owner = NULL;
+	HeapTuple 			tuple;
+	Form_sysdatabases 	sysdb;
+
+	tuple = SearchSysCache1(SYSDATABASENAME, CStringGetTextDatum(dbname));
+
+	if (!HeapTupleIsValid(tuple))
+		return InvalidDbid;
```

Comment:
I feel it doesn't make much sense to return InvalidDbid here. We can consider to throw an error if the given db name is invalid. And please add some comment ahead of this function saying that caller is responsible for validating db name before calling this function.